src/psyclone/tests/psyir/symbols/symbol_table_test.py

```diff
@@ -2452,6 +2452,7 @@ def test_rename_codeblock_error(fortran_reader):
 
 # resolve_imports
 
+@pytest.mark.usefixtures("clear_module_manager_instance")
```

Comment:
Oh, that's an interesting idea. However, doesn't `scope=class` mean that all tests in a class will *share* the fixture rather than running it for each one as we need? In the past, I've created a new fixture that simply makes use of the fixture that I want to `autouse` and then marked my new fixture as `autouse` for all test functions in that module.

Reply:
I admit I am not entirely certain how fixtures works in test classes. To be investigated at some time in the future :)